htdocs/societe/class/societe.class.php

```diff
@@ -1676,7 +1676,7 @@ public function fetch($rowid, $ref = '', $ref_ext = '', $barcode = '', $idprof1
 
 		$sql .= ' WHERE s.entity IN ('.getEntity($this->element).')';
 		if ($rowid) {
-			$sql .= ' AND s.rowid = '.((int) $rowid);
+			$sql .= " AND s.rowid = '" . $rowid . "'";
```

Comment:
rowid is an int. Why usign quotes for string ?
Old code seems good.

Reply:
The problem is that I have made my change before this commit (int) and I have forgoten to push my commit...